zuul.d/projects.yaml

```diff
@@ -118,15 +118,12 @@
         - cluster-api-provider-openstack-image-build:
             branches: master
 
-#NOTES: Should be updated when switch to official project
 - project:
-    name: zhongjun2/bosh-huaweicloud-cpi-release
+    name: cloudfoundry-incubator/bosh-huaweicloud-cpi-release
     check:
-      jobs:
-        - bosh-huaweicloud-cpi-release-acceptance-test
-    check-generic-cloud:
```

Comment:
I think we should use `check-generic-cloud`  to manual trigger, avoid creating lots of testing resource in public cloud.